src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -9301,7 +9311,10 @@ instruct compareAndExchangeL(iRegLNoSp res, indirect mem, iRegL oldval, iRegL ne
   ins_pipe(pipe_slow);
 %}
 
+// This encoding class is generated automatically from cas.m4.
+// DO NOT EDIT ANYTHING IN THIS SECTION OF THE FILE
```

Comment:
I didn't mean "exactly like". This is a pattern, not an encoding class. Sorry. :-)

Reply:
Er, yes, I wasn't thinking...